recipes/cookbook-uploader.rb

```diff
@@ -0,0 +1,36 @@
+#
+# Cookbook Name:: osl-jenkins
+# Recipe:: cookbook-uploader
+#
+# Copyright (C) 2015 Oregon State University
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+secrets = Chef::EncryptedDataBagItem.load(node['osl-jenkins']['databag'], 'secrets')
+repos = collect_github_repositories(secrets, 'osuosl-cookbooks')
```

Comment:
As `secrets` here is the databag item: `node['osl-jenkins']['databag']:secrets` I'm thinking it would contain more than a github token (including things like `id` and `name`). Shouldn't this be `secrets['github-token']` or something similar?